cypress/integration/login-test.js

```diff
@@ -0,0 +1,130 @@
+/* eslint-disable no-undef */
+const EMAIL = '';
+const PASSWORD = '';
```

Comment:
to possibly come from env var or some other config specification for automation etc

Reply:
100% should be injected from ENV